src/content/it/fundamentals/performance/rendering/index.md

```diff
@@ -0,0 +1,151 @@
+regole, vengono applicate e vengono calcolati gli stili finali per ogni
+elemento.
+* **Layout**. Una volta che il browser conosce quali regole si applicano
+ad un elemento può cominciare a calcolare quanto spazio occupa e dove
+sullo schermo. Il modello di layout web significa che un elemento può
+influenzare gli altri, ad esempio la larghezza dell'elemento `body` in
+genere colpisce le larghezze dei suoi figli e così via fino alla fine
+dell'albero, in modo che il processo possa essere abbastanza complesso
+per il browser.
+* **Paint**. Il Painting è il processo di riempimento di pixel. Esso
+implica il disegno di testo, colori, immagini, bordi e ombre,
+sostanzialmente ogni parte visiva degli elementi. Il disegno è
+tipicamente eseguito su superfici multiple, spesso denominate layer.
+* **Compositing**. Poiché le parti della pagina sono state disegnate in
+livelli potenzialmente multipli, devono essere disegnate sullo schermo
+nell'ordine corretto in modo che la pagina sia corretta. Ciò è
+particolarmente importante per gli elementi che si sovrappongono tra
+loro, poiché un errore potrebbe causare un elemento sovrapposto in
+maniera errata ad un altro.
+
+Ognuna di queste parti della pipeline rappresenta un'occasione per
+introdurre jank, quindi è importante capire esattamente quali parti
+della pipeline il codice innesca.
+
+A volte puoi sentire utilizzare il termine "rasterize" in
+combinazione con il paint. Questo perché la pittura è in realtà due
+compiti: 1) creare un elenco delle chiamate al disegno e 2) riempimento
+dei pixel.
+
+Quest'ultimo è chiamato "rasterization" e così ogni volta che vedi paint
+record in DevTools, dovresti pensarli come inclusivi di rasterizzazione.
+(In qualche architettura la creazione delle chiamate al disegno e
+rasterizing vengono eseguite diversi thread, ma non è qualcosa sotto il
+controllo degli sviluppatori.)
+
+Non dovrete sempre toccare ogni parte della pipeline per ogni frame.
+Infatti ci sono tre modalità in cui la pipeline _normalmente_ entra in
+causa per un dato frame quando si effettua una modifica visiva, sia con
+JavaScript, CSS o Web Animations:
+
+### 1. JS / CSS > Style > Layout > Paint > Composite
+
+<img src="images/intro/frame-full.jpg"  alt="La completa pixel pipeline">
+
+Se si modifica una proprietà "layout" con uno dei cambiamenti della
+geometria di un elemento come la sua larghezza, altezza, o posizione con
+sinistra o superiore, il browser dovrà controllare tutti gli altri
+elementi ed eseguire il "reflow" della pagina. Qualsiasi area
+interessata dovrà eseguire il re-paint e gli elementi finali disegnati
+dovranno essere composti nuovamente insieme.
+
+### 2. JS / CSS > Style > Paint > Composite
+
+<img src="images/intro/frame-no-layout.jpg" alt="La pixel pipeline senza layout.">
+
+Se si modifica una proprietà "paint only" come un'immagine di sfondo, il
+colore del testo o ombre, in altre parole che non influenza il layout
```

Comment:
colore del testo o le ombre, in altre parole qualcosa che non influenza il layout